pkg/storage/unified/resource/server.go

```diff
@@ -927,6 +927,9 @@ func (s *server) Search(ctx context.Context, req *SearchRequest) (*SearchRespons
 	if err := s.Init(ctx); err != nil {
 		return nil, err
 	}
+	if s.index == nil {
+		return nil, fmt.Errorf("search index not configured")
```

Comment:
In main, running `make run` I hit this case... do you see the same thing @scottlepp 

<img width="713" alt="image" src="https://github.com/user-attachments/assets/f6772573-304e-46c8-99a7-38141367ec26">